src/object_gui.cpp

```diff
@@ -321,6 +310,13 @@ class BuildObjectWindow : public Window {
 				break;
 			}
 
+			case WID_BO_OBJECT_NAME: {
+				const ObjectSpec *spec = ObjectClass::Get(_selected_object_class)->GetSpec(_selected_object_index);
+				StringID str = spec != nullptr ? spec->name : STR_EMPTY;
+				DrawStringMultiLine(r, str, TC_ORANGE, SA_CENTER);
```

Comment:
```suggestion
				if (spec != nullptr) DrawStringMultiLine(r, spec->name, TC_ORANGE, SA_CENTER);
```
Why go through the string drawing, when we know it's an empty string?